runtime-modules/working-group/src/benchmarking.rs

```diff
@@ -586,7 +586,7 @@ benchmarks_instance! {
 
         let (opening_id, successful_application_ids, _) =
             add_opening_and_apply_with_multiple_ids::<T, I>(
-                &(1..i).collect::<Vec<_>>(),
+                &(1..i+1).collect::<Vec<_>>(),
```

Comment:
```suggestion
                &(1..=i).collect::<Vec<_>>(),
```
https://doc.rust-lang.org/rust-by-example/flow_control/for.html

Reply:
Fixed here: https://github.com/Joystream/joystream/pull/3486/commits/35c906aaa7f75c133182bfd49b509148ab8d6712